pkg/antctl/raw/multicluster/leave.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package multicluster
+
+import (
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"antrea.io/antrea/pkg/antctl/raw/multicluster/common"
+)
+
+var leaveOpts *common.CleanOptions
+
+var leaveExamples = strings.Trim(`
+# Leave the ClusterSet.
+  antctl mc leave --namespace kube-system --clusterset test-clusterset
```

Comment:
Can we leave Namespace as `kube-system` by default?  then we can make only one --clusterset required.

Reply:
Set `kube-system` as default now.